app/notebooks/question_answer_generator/q_a_generation.ipynb

```diff
@@ -41,33 +69,129 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 19,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "2624808f256c43e793bb61d7a6ae3aa8",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "RadioButtons(description='Question source:', options=('document', 'chunk'), value='document')"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "question_gen_selection = widgets.RadioButtons(\n",
+    "    options=['document', 'chunk'],\n",
+    "    value='document',\n",
+    "    description='Question source:',\n",
+    "    disabled=False\n",
+    ")\n",
+    "display(question_gen_selection)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "df7a2ace17ba401aabdf124a024373e0",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "SelectMultiple(description='Dataset source:', index=(0, 1), options=('Imagine LA', 'CA EDD', 'BEM'), value=('I…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "question_gen_selection = input(\"Generate questions by chunk or document?\")"
+    "question_dataset = widgets.SelectMultiple(\n",
+    "    options=['Imagine LA', 'CA EDD', 'BEM'],\n",
+    "    value=['Imagine LA', 'CA EDD' ],\n",
```

Comment:
```suggestion
    "    value=['Imagine LA', 'CA EDD', 'BEM' ],\n",
```

Reply:
Oops I misunderstood, no, the code as you've written makes sense to me in that case!